add_progress.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>PROGRESS TRACKER</title>
+	<link rel="stylesheet" type="text/css" href="main.css">
+</head>
+<body>
+	<h1>TRACK & ADD YOUR PROGRESS</h1>
+
+	<!--THIS IS THE ADD PROGRESS PAGE, USERS ENTER THE WORKOUT TYPE THEY PLAYED AND DATE, THEY CAN ALSO PRESS ON THE TRACK PROGRESS BUTTON WHICH REDIRECTS THEM TO CHECK OUT THEIR WORKOUTS-->
+
```

Comment:
The page form has been updated to match the theme and be consistent with the user interface of other pages. Contains the necessary fields to enter the workout information.